python-checks/src/test/java/org/sonar/python/checks/ExpressionsTest.java

```diff
@@ -83,6 +88,85 @@ public void remove_parentheses() {
     assertThat(removeParentheses(null)).isNull();
   }
 
+  @Test
+  public void unescape_string_literal() {
+    assertThat(unescape(stringLiteral("''"))).isEqualTo("");
+    assertThat(unescape(stringLiteral("'' ''"))).isEqualTo("");
+    assertThat(unescape(stringLiteral("'abc' 'def'"))).isEqualTo("abcdef");
+  }
+
+  @Test
+  public void unescape_string_element() {
```

Comment:
This test is very long. Can we split it with meaningful names for the new tests?